src/app/services/hris/employee/employee-work-experience.service.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { environment } from '../../../../environments/environment';
+import { WorkExperience } from 'src/app/models/hris/work-experience.interface';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WorkExperienceService {
+
+    baseUrl: string;
+    
+    constructor(private httpClient: HttpClient) { 
+        this.baseUrl =`${environment.HttpsBaseURL}/work-experience`
+    }
+
+    getWorkExperience(id: number): Observable<WorkExperience[]> {
+        return this.httpClient.get<WorkExperience[]>(`${this.baseUrl}?id=${id}`)
+      }
+
+    save(experience: WorkExperience): Observable<WorkExperience> {
+        return this.httpClient.post<WorkExperience>(`${this.baseUrl}`, experience);
+    }
+
+    update(experience: WorkExperience): Observable<any> {
```

Comment:
maybe try and name the methods ex. saveWorkexperience

Reply:
made the names more descriptive